canonical/snapcraft.io
# Update discourse version

## Done
- Update discourse version from 5.4.8 to 5.4.9
## How to QA
- Go to https://snapcraft-io-4504.demos.haus/docs/installing-snap-on-manjaro-linux and ensure that the image displays
## Issue / Card
Fixes #

## Screenshots
